src/lib/unpackPlotProps.js

```diff
@@ -6,7 +6,7 @@ export default function unpackPlotProps(props, context) {
   const {container, getValObject, defaultContainer, updateContainer} = context;
 
   if (!props.attr) {
-    throw new Error('connectedToContainer components require an `attr` prop');
+    return {};
```

Comment:
this change required so that `<Section>` without `attr` wouldn't throw, and it's why we deleted the test above.

Reply:
yeah, ok, i've been needing unpackPlotProps to not require an attr as well, so ok